frontend/src/Board.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import ReactDOM from "react-dom";
+import './Board.css';
+
+const Board = (props) => {
+  return (
+    <div className='board'>
+      <div className='board__header'>
+        <div className='board__header__title'>
+          <div className='board__header__title__image'>
+            {props.image}
+          </div>
+          <div className='board__header__title__text'>
+            <h1>{props.title}</h1>
+          </div>
+          <div className='board__header__title__menu'>
+            <span className='menu__dot'></span>
+            <span className='menu__dot'></span>
+            <span className='menu__dot'></span>
```

Comment:
Streść to do jednego tagu jesli dasz radę. Wyeksportuj może też ten przycisk do osobnego komponentu?